.github/.kube-linter-config.yaml

```diff
@@ -1,13 +1,15 @@
 
 checks:
+  ignorePaths:
+#    - "/home/runner/work/konflux-ci/konflux-ci/.kube-linter/out-dependencies-keycloak-deployment.yaml"
```

Comment:
Is this a leftover?

Reply:
It was commented because i wanted to test that the patches work, once I saw they did I deleted this line (and all `IgnorePaths` actually)